<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog_results</class>
 <widget class="QDialog" name="Dialog_results">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>330</width>
    <height>551</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Results</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#groupBox {
	font: 87 8pt &quot;Arial Black&quot;;
	color: rgb(114, 114, 114);
 }

#groupBox_2 {
	font: 87 8pt &quot;Arial Black&quot;;
	color: rgb(114, 114, 114);
 }

#lineEdit_gt{ 
	background-color: rgb(227, 227, 227);
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Filtering</string>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Model:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="label_Modelo">
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>Face detector:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLabel" name="label_detector">
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Filter algorithm:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLabel" name="label_TFiltro">
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Number of particles:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLabel" name="label_NoParticulas">
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_resAlgorithm1">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Resampling algorithm:</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QLabel" name="label_resAlgorithm">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>-----------</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="label_resPercent1">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Resampling percent:</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QLabel" name="label_resPercent">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>-----------</string>
        </property>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Estimation algorithm:</string>
        </property>
       </widget>
      </item>
      <item row="6" column="1">
       <widget class="QLabel" name="label_TSalida">
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
      <item row="7" column="0">
       <widget class="QLabel" name="label_robustMean1">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Robust mean percent:</string>
        </property>
       </widget>
      </item>
      <item row="7" column="1">
       <widget class="QLabel" name="label_robustMean">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>-----------</string>
        </property>
       </widget>
      </item>
      <item row="8" column="0">
       <widget class="QLabel" name="label_10">
        <property name="text">
         <string>State Space:</string>
        </property>
       </widget>
      </item>
      <item row="8" column="1">
       <widget class="QLabel" name="label_ee">
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
      <item row="9" column="0">
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>Observation model:</string>
        </property>
       </widget>
      </item>
      <item row="9" column="1">
       <widget class="QLabel" name="label_obsmodel">
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
      <item row="11" column="0">
       <widget class="QLabel" name="label_11">
        <property name="text">
         <string>Elapsed time:</string>
        </property>
       </widget>
      </item>
      <item row="11" column="1">
       <widget class="QLabel" name="label_t">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="12" column="0">
       <widget class="QLabel" name="label_12">
        <property name="text">
         <string>Aprox. FPS:</string>
        </property>
       </widget>
      </item>
      <item row="12" column="1">
       <widget class="QLabel" name="label_fps">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="10" column="0">
       <widget class="QLabel" name="label_runs">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>Algorithm runs:</string>
        </property>
       </widget>
      </item>
      <item row="10" column="1">
       <widget class="QLabel" name="label_resultRuns">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>1</string>
        </property>
       </widget>
      </item>
     </layout>
     <zorder>label</zorder>
     <zorder>label_Modelo</zorder>
     <zorder>label_6</zorder>
     <zorder>label_detector</zorder>
     <zorder>label_2</zorder>
     <zorder>label_TFiltro</zorder>
     <zorder>label_5</zorder>
     <zorder>label_NoParticulas</zorder>
     <zorder>label_resAlgorithm1</zorder>
     <zorder>label_resAlgorithm</zorder>
     <zorder>label_4</zorder>
     <zorder>label_TSalida</zorder>
     <zorder>label_11</zorder>
     <zorder>label_t</zorder>
     <zorder>label_12</zorder>
     <zorder>label_fps</zorder>
     <zorder>label_7</zorder>
     <zorder>label_obsmodel</zorder>
     <zorder>label_10</zorder>
     <zorder>label_ee</zorder>
     <zorder>label_resPercent1</zorder>
     <zorder>label_resPercent</zorder>
     <zorder>label_robustMean1</zorder>
     <zorder>label_robustMean</zorder>
     <zorder>label_runs</zorder>
     <zorder>label_resultRuns</zorder>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>143</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="PB_error">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Show Error</string>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="PB_acept">
       <property name="text">
        <string>Accept</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
